### PR TITLE
Document profiler overhead

### PR DESCRIPTION
**Pull request by 94noni**
After upgrading the bundle recipe on a local app
I saw the new profiler icon and also some queries overhead (~10queries)

Ref https://github.com/symfony/recipes/blob/9a7f5bcebbda2f5244c4be4133a6bd181080cbea/doctrine/doctrine-migrations-bundle/3.1/config/packages/doctrine_migrations.yaml#L6

### CODE REVIEW

File: Resources/doc/index.rst [hunk 1]
```diff
@@ -83,6 +83,9 @@ application:
         # Whether or not to wrap migrations in a single transaction.
         transactional: true
 
+        # Whether or not to enable the profiler collector. This add some queries overhead.
```
Comment (greg0ire) [inline Resources/doc/index.rst]:
```suggestion
        # Whether or not to enable the profiler collector. This adds some queries overhead.
```

**Review by greg0ire — COMMENTED** [review]
I think this should be put in sync with https://github.com/doctrine/DoctrineMigrationsBundle/blob/453b0979579c4ab214a37b28d0b3cc8b471ac072/DependencyInjection/Configuration.php#L150-L153

File: Resources/doc/index.rst (hunk 1 above)
Comment (94noni) [reply Resources/doc/index.rst]: @greg0ire indeed I can use the same sentence `Use profiler to calculate and visualize migration status.`

do I need to target v3.1? and keep there value `false`

thx

Comment (greg0ire) [reply Resources/doc/index.rst]: The correct branch to target is 3.2, as the last release is 3.2.2

**Comment by greg0ire** [comment]
> Update index.rst

You are going to edit several files, so might as well use something higher level, like "Document profiler overhead"

**Comment by SenseException** [comment]
> You are going to edit several files

I don't understand what the scope is for this PR.

**Comment by greg0ire** [comment]
IMO it should be:
- Resources/doc/index.rst
- DoctrineMigrationsBundle/DependencyInjection/Configuration.php

That way we have some sync between both files.

**Comment by 94noni** [comment]
the main goal is to document this configuration that is not present in the https://symfony.com/bundles/DoctrineMigrationsBundle/current/index.html

the current readme change can be reverted and if accepted, done in another PR

**Comment by greg0ire** [comment]
> the current readme change can be reverted and if accepted, done in another PR

It's fine to me, but to make things perfectly clear, I would expect your addition about the overhead to appear in `DoctrineMigrationsBundle/DependencyInjection/Configuration.php` as well. Right now I no longer even see it in the diff. All I see is that you synced `Resources/doc/index.rst` which is great, but now I'm waiting for you to add your sentence about the overhead in both files.

**Comment by 94noni** [comment]
@greg0ire it is absolutely not what I have understood reading (and rereading) your comments
sorry about that, I will rewrite soon with what I had in mind first

**Comment by 94noni** [comment]
@greg0ire hi, here are the changes I propose, thx!

**Review by greg0ire — APPROVED** [review]
To be squash merged

**Comment by greg0ire** [comment]
Thanks @94noni !

**Pull request merged**